docker-compose.yaml

```diff
@@ -1,17 +1,24 @@
 version: '3.4'
 
 services:
+    db:
+        image: postgres:11.2-alpine
     app:
+        &app
         build:
             context: .
             target: prod
         environment:
             APP_SECRET:
             DEFAULT_LOCALE:
             SERVICE_NAME:
+            DATABASE_URL: pgsql://postgres:@db:5432/postgres
         image: libero/content-store:${IMAGE_TAG:-master}
-        volumes:
-            - ./.docker/data:/app/data/
+        depends_on:
+            - db
+    db-schema:
+        <<: *app
+        command: bin/console doctrine:migrations:migrate --no-interaction
```

Comment:
Appears not to have to wait on `db` being initialised.

Reply:
Was it luck?
Adding to `postgres`:
```
command: bash -c "sleep 10 && postgres" 
```
results in:
```
db-schema_1  | In AbstractPostgreSQLDriver.php line 73:
db-schema_1  |                                                                                
db-schema_1  |   An exception occurred in driver: SQLSTATE[08006] [7] could not connect to s  
db-schema_1  |   erver: Connection refused                                                    
db-schema_1  |          Is the server running on host "db" (172.23.0.2) and accepting               
db-schema_1  |          TCP/IP connections on port 5432?        
```